meteor-app/imports/api/parts/parts.js

```diff
@@ -0,0 +1,44 @@
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from  'simpl-schema'
+
+import { createdAt, updatedAt } from '/imports/api/schema'
+
+const Parts = new Mongo.Collection('parts')
+
+export const PartsSchema = new SimpleSchema({
+  imageUrl: {
+    type: String,
+    defaultValue: '/public/images/logo-large.jpg',
+  },
+  retailPrice: {
+    type: Number,
```

Comment:
Please use `SimpleSchema.Integer` instead of `Number` for this and all places where you want an integer rather than a floating point value